app/src/main/java/org/mozilla/focus/utils/FeatureFlags.kt

```diff
@@ -7,5 +7,5 @@ package org.mozilla.focus.utils
  * Feature flag for MVP
  */
 object FeatureFlags {
-    const val isMvp = false
+    const val isMvp = true
```

Comment:
Are we ready to flip this flag? Maybe to make it easier for testing, we could use:

```suggestion
    const val isMvp =  BuildConfig.BUILD_TYPE.contains("debug")
```